grafana/grafana-experimental
# OperationEditor: Fix rendering of operation editor items

I have introduced an issue when I was refactoring OperationEditor (at that time in `grafana/grafana` , but because I wasn't passing props correctly to component, it was doing a lot of mounting - unmounting https://github.com/grafana/grafana/commit/7be8301a265c7b228d8cee77188a094504fa1335. I will have to do this fix also in `grafana/grafana`

This PR takes previously called `StyledOperationHeader` and moves it to separate file and makes it a proper component that doesn't cause -remounting of each operation editor. 

Fixed:

https://github.com/grafana/grafana-experimental/assets/30407135/168a846f-ceaa-43f5-9236-35f298d6b754

Broken:

https://github.com/grafana/grafana-experimental/assets/30407135/7c5b7724-ab1c-4b59-bde3-3b0769b9676d

How to test:
- If you can use `yarn link` and test this locally, do that.
- For me it works only by:
  -  1. renaming package in `package.json` to `custom`
  - in `grafana/grafana` `package.json` add `"custom": "file: ../grafana-experimental"`
  - import `OperationList` in Loki from `custom` 
  - check the recording and do that/other interactions
